apps/agent/scripts/ganache-cli.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
I don't think there's much need for this, but perhaps I've missed something.

1. Perhaps it would be worth using the ganache plugin directly? I was trying this, and while it doesn't seem to fix the failing tests, it allows us to remove this script :).
2. I'm still wondering if we shouldn't just use buidlerevm by default; it appears to run much much faster on tests (and the only failures are due to revert issues that we should fix in `contract-helpers-test`)
3. The solidity coverage plugin appears to work well out of the box, and it controls starting and stopping ganache itself (e.g. running `npx buidler coverage --network coverage` seems to just work)

Reply:
No, you are right. Last time I checked (recently) it was not possible and was handle with a similar script than ours. Now the tools do the heavy lift themselves or use buidlerevm 🙌